src/ghdcbot/engine/snapshots.py

```
@@ -279,20 +276,18 @@ def _collect_snapshot_data(
+    recent_notifications = storage.list_recent_notifications(limit=1000)
+    for notif in recent_notifications:
+        notifications_data.append({
+            "dedupe_key": notif.get("dedupe_key"),
+            "event_type": notif.get("event_type"),
+            "github_user": notif.get("github_user"),
+            "discord_user_id": notif.get("discord_user_id"),
+            "repo": notif.get("repo"),
+            "target": notif.get("target"),
+            "channel_id": notif.get("channel_id"),
+            "sent_at": notif.get("sent_at"),
+        })
```

Ruff (0.15.7)

[warning] Use a list comprehension to create a transformed list

(PERF401)